app/controllers/concerns/document_attachable.rb

```diff
@@ -25,6 +25,10 @@ def document_provided?
     tribunal_case&.documents(document_key)&.any? || document.present?
   end
 
+  def document_presence
```

Comment:
I'm not sure if this is the right name for this method. `document_presence` sounds like a method that would return a boolean, depending on whether a document was present or not. I would change this to `validate_document_presence`